### PR TITLE
fix(VAutocomplete): divider/subheader not rendering

### LINKED ISSUE #15721: [Bug Report][3.0.0] v-autocomplete items header / divider not working
### Environment
**Vuetify Version:** 3.0.0-beta.9
**Vue Version:** 3.2.38
**Browsers:** Chrome 104.0.0.0
**OS:** Windows 10

### Steps to reproduce
This feature does not work in your example.

### Expected Behavior
v-autocomplete reads the options in the object.
image

### Actual Behavior
Just empty space
image

### Reproduction Link
<a href="https://codepen.io/ppyl-datboi/pen/dyebexE?editors=101" target="_blank">https://codepen.io/ppyl-datboi/pen/dyebexE?editors=101</a>

### PR DESCRIPTION
**Pull request by nekosaur**
## Description

closes #15721

## Markup:

<details>

```vue
<template>
  <div>
    <v-select
      :items="people"
      filled
      chips
      closable-chips
      color="blue-grey-lighten-2"
      label="Select"
      multiple
    ></v-select>

    <v-autocomplete
      :items="people"
      filled
      chips
      closable-chips
      color="blue-grey-lighten-2"
      label="autocomplete"
      multiple
    ></v-autocomplete>

    <!-- <v-list :items="people" /> -->
  </div>
</template>

<script>
  const srcs = {
    1: 'https://cdn.vuetifyjs.com/images/lists/1.jpg',
    2: 'https://cdn.vuetifyjs.com/images/lists/2.jpg',
    3: 'https://cdn.vuetifyjs.com/images/lists/3.jpg',
    4: 'https://cdn.vuetifyjs.com/images/lists/4.jpg',
    5: 'https://cdn.vuetifyjs.com/images/lists/5.jpg',
  }

  export default {
    data: () => ({
      people: [
        {
          title: 'Group 1',
          subheader: true,
          divider: true,
          children: [
            { title: 'Jane Smith', group: 'Group 1', avatar: srcs[5] },
            { title: 'Britta Holt', group: 'Group 1', avatar: srcs[4] },
          ],
        },
        {
          title: 'Group 2',
          subheader: true,
          children: [
            { title: 'Trevor Hansen', group: 'Group 2', avatar: srcs[3] },
            { title: 'Ali Connors', group: 'Group 2', avatar: srcs[2] },
          ],
        },
      ],
    }),
  }
</script>



```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [ ] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by nekosaur** [comment]
- type is a very bad prop name, needs to be renamed
- I kept item type `divider` because I didn't want to add another `itemX` prop and special parsing for it

**Comment by paul-thebaud** [comment]
> How about groups instead so they can be kept while filtering?

Hi, I agree with @KaelWD about real groups inside v-select/autocomplete, because the current implementation is not accessibility friendly. See the [group ARIA role doc](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/group_role).

**Comment by volarname** [comment]
+1 for merge

**Comment by MatthewAry** [comment]
Looking forward to this being resolved. This is currently blocking my move from Vuetify 2 → 3. Thanks!

File: packages/vuetify/src/components/VSelect/VSelect.tsx [hunk 1]
```diff
@@ -10,26 +10,26 @@ import { VDialogTransition } from '@/components/transitions'
 import { VList, VListItem } from '@/components/VList'
 import { VMenu } from '@/components/VMenu'
 import { VTextField } from '@/components/VTextField'
+import { VListChildren } from '../VList/VListChildren'
```
Comment (johnleider) [inline packages/vuetify/src/components/VSelect/VSelect.tsx:13]:
```suggestion
import { VListChildren } from '@/components/VList/VListChildren'
```

**Review by johnleider — DISMISSED** [review]

**Comment by nekosaur** [comment]
> How about groups instead so they can be kept while filtering?

This was a nice suggestion, until virtual scroll came along :|

**Comment by func0der** [comment]
This is closed without a fix getting to master or am I just blind?

**Pull request closed without merging**